test/wolff_tests.erl

```diff
@@ -476,11 +476,15 @@ check_connectivity_test() ->
   ClientCfg = client_config(),
   {ok, Client} = start_client(ClientId, ?HOSTS, ClientCfg),
   ?assertEqual(ok, wolff:check_connectivity(ClientId)),
+  ?assertEqual(ok, wolff:check_connectivity(?HOSTS, ClientCfg, <<"test-topic">>)),
+  ?assertEqual(
```

Comment:
`wolff:check_connectivity/3` can return 3 things:
1) `ok`
   - if at least one of the hosts is connected and it has the topic created;
1) `{error, unknown_topic_or_partition}`
   - if at least one of the hosts is connected and it doesn't have the topic created;
1) `{error, List}`
   - if no hosts are connected. In this case, `List` is a list of maps `#{host => Host, reason => Reason}`.

In this specific test, as host is connected, it will only return `ok` or `{error, unknown_topic_or_partition}`.